libs/interpreter-lib/project.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "interpreter-lib",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "libs/interpreter-lib/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "external": "none", // bundle dependencies to make usable externally
```

Comment:
JSON does not know comments.

Reply:
Done in [f76ae1f](https://github.com/jvalue/jayvee/pull/388/commits/f76ae1f472c299da41d4352532f9100f31b7a0aa)